docs/en/glossary/glossary.asciidoc

```diff
@@ -69,6 +69,17 @@ necessary to perform an analytics task. See
 include::{es-repo-dir}/glossary.asciidoc[tag=api-key-def]
 --
 
+[[glossary-apm-agent]] APM agent::
+An open-source library, written in the same language as your service,
+which <<glossary-instrumentation,instruments>> your code and collects performance data
+and errors at runtime.
+//Source: Observability
+
+[[glossary-apm-server]] APM Server::
+An open-source application that receives data from <<glossary-apm-agent,APM agents>> and transforms
```

Comment:
Not sure we want to use the word "transforms" here since we have the concept of a [transform](https://www.elastic.co/guide/en/elasticsearch/reference/7.9/transform-overview.html) in Elasticsearch. Can we say that it "sends the data to {es}" or "stores the data in {es}" instead?